Meadowlark-js/package.json

```diff
@@ -20,22 +20,22 @@
     "@aws-sdk/smithy-client": "3.41.0",
```

Comment:
I can't add a comment on line 3... can you bump the minor version?

Reply:
I made it 0.2.0-dev.0